config/issuetracker.go

```diff
@@ -43,6 +46,7 @@ var originPatterns = map[IssueTracker]*regexp.Regexp{
 	IssueTrackerPivotal:  regexp.MustCompile(`^(https?://)?(www\.)?pivotaltracker\.com/n/projects/[0-9]+`),
 	IssueTrackerRedmine:  regexp.MustCompile(`^(https?://)?[a-zA-Z0-9\-]+(\.[a-zA-Z0-9]+)+(:[0-9]+)?$`),
 	IssueTrackerYoutrack: regexp.MustCompile(`^(https?://)?(www\.)?[0-9A-z-]{2,}\/?.*$`),
+	IssueTrackerAzure:    regexp.MustCompile(`^(https?://)?[a-zA-Z0-9\-.]+/([a-zA-Z0-9]+)+/([a-zA-Z0-9]+)+.*$`),
```

Comment:
Can Azure boards be hosted on-prem? Isn't it always available under the azureboards domain?

Reply:
I think that they couldn't be. Checked again but haven't found any evidence about that. Should I drop the regex validation here?

Reply:
You can change the validation to make sure that the host is whatever azureboards' domain is. See for example pivotaltracker regex.